### PR TITLE
💄 [nft/_iscnId] Add Writing NFT Preview Card

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2022-10-16 下午4 39 50

### CODE REVIEW

File: components/NFTPreviewCard.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      <div
+        :class="[
+          'flex',
+          'flex-col',
+          'w-full',
+          'rounded-[8px]',
+          'overflow-hidden',
+        ]"
+      >
+        <img id="imgElement" alt="og image" src="" />
```
Comment (williamchong) [inline components/NFTPreviewCard.vue:51]: Why not use prop or ref?

File: pages/nft/iscn/_iscnId.vue [hunk 2]
```diff
@@ -414,6 +431,8 @@ export default class NFTTestMintPage extends Vue {
       logTrackerEvent(this, 'IscnMintNFT', 'GetOgImageExists', url, 1);
       const { data } = await this.$axios.get(`/crawler/ogimage?url=${encodeURIComponent(url)}`, { responseType: 'blob' })
       this.ogImageBlob = data
+      const objectURL = URL.createObjectURL(data);
+      this.imgSrc = objectURL
```
Comment (williamchong) [inline pages/nft/iscn/_iscnId.vue:435]: So `imgSrc` can be a `computed` base on `this.ogImageBlob`?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: components/icon/Eye.vue [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <svg
+    width="20"
+    height="14"
```
Comment (nwingt) [inline components/icon/Eye.vue:4]: Should be a square?

**Review by williamchong — APPROVED** [review]

File: components/icon/Eye.vue [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <template>
   <svg
     width="20"
-    height="14"
+    height="20"
     viewBox="0 0 20 14"
```
Comment (nwingt) [inline components/icon/Eye.vue:5]: Should change viewBox and path too I suppose

**Review by nwingt — APPROVED** [review]

**Pull request merged**